### ISSUE TITLE
noclip possible accuracy

### ISSUE THREAD

**Issue by eckshton** [absolllute/Mega-Hack-Pro-Future#1137]
instead of killing you when you fall below the set accuracy it'd be cool if it could kill you when you fall below the max possible accuracy. so megahack would figure out the number of frames in a level on startup and use that instead of frames passed. pretty sure they have this in osu.

**Comment by absoIute** [comment]
so like a frames missed death trigger for Noclip Accuracy?

**Comment by eckshton** [comment]
yeah. instead of how it currently is ((frames-dead frames) / frames) it would be ((frames - dead frames) / total frames)
